frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -183,10 +199,12 @@ const EditRecipeForm = ({ history }) => {
         ingredients={ingredients}
         setIngredients={setIngredients}
       />
+      <div className="errorMsg">{ingredients.length <= 1 ? ingredientsError : ''}</div>
       <EditDirections
         directions={directions}
         setDirections={setDirections}
       />
+      <div className="errorMsg">{directions.length <= 1 ? directionsError : ''}</div>
```

Comment:
I think you can just do `{directionsError}`, because the `directionsError ` is an empty string by default.